.github/CONTRIBUTING.md

```diff
@@ -9,66 +9,58 @@ git clone https://github.com/transloadit/uppy.git
 cd uppy
 ```
 
-We are using [Corepack][] to manage version of [Yarn][]. Corepack comes pre-installed with Node.js >=16.x, or can be installed through `npm`. You can run `corepack enable` to install a `yarn` executable in your `$PATH`, or prefix all yarn commands with `corepack yarn`.
+We are using [Corepack][] to manage versions of [Yarn][].\
+Corepack comes pre-installed with Node.js >=16.x, or can be installed through `npm`:
 
 ```sh
 corepack -v || npm i -g corepack
-yarn -v || corepack enable
-yarn install || corepack yarn install
 ```
 
-[Corepack]: https://nodejs.org/api/corepack.html
+You can either run `corepack enable` to install a `yarn` executable in your `$PATH`, and run all `yarn` commands in this guide as they are written:
 
-[Yarn]: https://yarnpkg.com/
+```sh
+corepack enable
+# e.g. `yarn install`
+```
 
-Our website’s examples section is also our playground, please read the [Local Previews](#Local-previews) section to get up and running.
+or prefix all `yarn` commands in this guide with `corepack yarn`:
 
-### Requiring files
+```sh
+# e.g. `corepack yarn install`
```

Comment:
IMO it's a bit weird to use a comment here.
```suggestion
corepack yarn install
```

Reply:
@aduh95, @arturi about this - take a look at the full file https://github.com/transloadit/uppy/blob/6dfdb87ec62229fa2215e5299edd2fb65944e883/.github/CONTRIBUTING.md

<img width="1023" alt="image" src="https://github.com/transloadit/uppy/assets/7578559/f15f475a-a4b5-44a8-b728-f81c52205bac">

Either both of these should be comments, or none of them.

I originally made both of them comments becaues we're just explaining how to use corepack & read our docs at the moment, we're not telling the reader to `yarn install` yet.

We're telling them to `yarn install` in the next section: https://github.com/transloadit/uppy/blob/6dfdb87ec62229fa2215e5299edd2fb65944e883/.github/CONTRIBUTING.md#development.

Reply:
If you think it's more intuitive I can make both of these commands (`yarn install` and `corepack yarn install`) & then remove the `yarn install` from the development section though